### PR TITLE
MP-4100: Adjust Marketplace Return Management feature IG.

### PR DESCRIPTION
**Pull request by sakharova-yuliia**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md [hunk 1]
```diff
@@ -1039,6 +1179,9 @@ Add product offers section to marketplace section of `navigation.xml`:
         </pages>
     </sales>
     <marketplace>
+        <label>Marketplace</label>
+        <title>Marketplace</title>
+        <icon>fa-shopping-basket</icon>
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md:1184]: the same

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md [hunk 2]
```diff
@@ -941,13 +1028,63 @@ class MerchantOmsCommunicationFactory extends SprykerMerchantOmsCommunicationFac
 
 </details>
 
+
+<details>
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md:1032]: why was it moved? Should be under `Set up widgets` section, please see https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides
According to this doc it was done in proper way with `Feature frontend` section

**Review by ilyakubanov — COMMENTED** [review]
- Please add prerequisites features, list of new modules and transfers will be adjusted in this case.

**Comment by ilyakubanov** [comment]
Prerequisites are missing

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md [hunk 3]
```diff
@@ -941,13 +1028,63 @@ class MerchantOmsCommunicationFactory extends SprykerMerchantOmsCommunicationFac
...
+<summary markdown='span'>src/Pyz/Yves/ShopApplication/ShopApplicationDependencyProvider.php</summary>
```
[the remark this answers is not in the document]
Comment (tobi812) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md:1033]: @ilyakubanov For all other plugins it was done in the same way. In the preview in PHPStorm it also works well, so the code-snippet will be collapsed. Question should I change it everywhere or keep it?

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md [hunk 4]
```diff
@@ -941,13 +1028,63 @@ class MerchantOmsCommunicationFactory extends SprykerMerchantOmsCommunicationFac
...
+
+{% info_block warningBox "Verification" %}
+
+<!--Describe how a developer can check they have completed the step correctly.-->
+
+Make sure that the following widgets have been registered by adding the respective code snippets to a Twig template:
+
+| WIDGET | VERIFICATION |
+| ---------------- | ----------------- |
+| MerchantSalesReturnCreateFormWidget | Go through the Return flow in the same way as now by clicking the **Create Return** button on the top of the *Order Details* page. Go to the *Create Return* page and create a return only with the items of one merchant order at a time and only for returnable items. |
```
[the remark this answers is not in the document]
Comment (tobi812) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md:1077]: @ilyakubanov It was already added in another section.

**Review by ilyakubanov — APPROVED** [review]

**Pull request merged**